backend/internal/controller/oidc_controller.go

```diff
@@ -160,22 +160,28 @@ func (oc *OidcController) createTokensHandler(c *gin.Context) {
 // @Security OAuth2AccessToken
 // @Router /oidc/userinfo [get]
 func (oc *OidcController) userInfoHandler(c *gin.Context) {
-	authHeaderSplit := strings.Split(c.GetHeader("Authorization"), " ")
-	if len(authHeaderSplit) != 2 {
+	_, authToken, ok := strings.Cut(c.GetHeader("Authorization"), " ")
```

Comment:
`strings.Split` allocates a slice in the heap and scans the entire string. `strings.Cut` is more efficient when we know there's at most 2 items, since it doesn't cause heap allocations and it stops at the first match